examples/with-passport/components/header.js

```diff
@@ -0,0 +1,57 @@
+import Link from 'next/link'
+
+const Header = () => (
+  <header>
+    <nav>
+      <ul>
+        <li>
+          <Link href="/">
+            <a>Home</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/login">
+            <a>Login</a>
+          </Link>
+        </li>
+        <li>
+          <Link href="/profile">
+            <a>Profile</a>
+          </Link>
+        </li>
+        <li>
+          <a href="/api/logout">Logout</a>
```

Comment:
I think you should use `const user = useUser()` and hide Logout if `user` doesn't exist.

Reply:
I wanted to avoid a layout switch there, in a real application you would end up adding a cool fade in like the one we have for zeit.co

Reply:
Quick note but when reading/using the with-cookies example, I too thought that usually the login/logout buttons would be shown/hidden based on if the user is logged in or not. It felt not complete.

I think it's fine if it's just a `{loggedIn && <li>..</li>}`, people will get it and at least they'll see the hooks in practice here.

thoughts?

Reply:
I think if we do something like this the layout change will be minimal?

## Logged out state: Home / Sign up / Login

<img width="693" alt="Screen Shot 2020-03-03 at 12 09 43 PM" src="https://user-images.githubusercontent.com/992008/75815261-f43e3480-5d47-11ea-9ac8-ab72d0a9037d.png">

## Logged in state: Home / Profile / Logout

<img width="675" alt="Screen Shot 2020-03-03 at 12 09 52 PM" src="https://user-images.githubusercontent.com/992008/75815324-0ae48b80-5d48-11ea-9998-ee2e9d3684d3.png">

Reply:
yes to me that more accurately represent a real-world use case